### ISSUE TITLE
Add/remove user roles with same command

### ISSUE THREAD

**Issue by dylandegrace** [Cog-Creators/Cog-Board#17]
* Description of cog:
Provide a command (similar to Dynobot's _role_ command) that would allow adding and removing of roles at the same time. Should be usable on roles with permissions with restrictions set by a server admin -- prevents a user adding the _admin_ role for instance.
* API's or other information for cog:
Example: `[p]role +Elite,-Basic` : Adds the role 'Elite' to the user and removes 'Basic'
* Other info (Payments, issues, ETC):
None

**Comment by palmtree5** [comment]
To be clear, anyone would be able to change their roles or would it need to be done by a mod or admin?

**Comment by dylandegrace** [comment]
Its utility would be that anyone could do this. The admin would set a list of ranks to be added/removed (e.g. `!Editrole set Streamer, gamer, EU, NA`) and then any user can add/remove from that list (e.g. `!editrole +streamer, -EU, +NA`).

I hope that I'm clear enough. Thanks!

**Comment by Kowlin** [comment]
hmm, in theory this looks interesting and would be fun to add on my list of "role management cogs" but however, first take a look at my ``buyrole``, see if its something you would like. or at least it's idea.

**Comment by dylandegrace** [comment]
Yeah, the buyrole works for _adding_ but it wouldn't work for removing (via a command). The addrole and removerole from (I think) Squids repo also work, except that you can't add and remove with a single command.
